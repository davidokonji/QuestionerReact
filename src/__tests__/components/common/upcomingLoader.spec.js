import { shallow } from 'enzyme';
import React from 'react';
import Loader from '../../../components/common/upcomingLoader';

describe('Upcoming Loader', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Loader />);

    expect(wrapper).toMatchSnapshot();
  });
});
