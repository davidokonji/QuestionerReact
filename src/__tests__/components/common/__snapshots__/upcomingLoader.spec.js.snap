// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upcoming Loader should render correctly 1`] = `
<div
  className="col-4 col-md-4"
>
  <ContentLoader
    height={160}
    primaryColor="#f3f3f3"
    secondaryColor="#ecebeb"
    speed={2}
    width={350}
  >
    <rect
      height="0"
      rx="0"
      ry="0"
      width="0"
      x="184"
      y="123"
    />
    <rect
      height="0"
      rx="0"
      ry="0"
      width="0"
      x="247"
      y="60"
    />
    <rect
      height="0"
      rx="0"
      ry="0"
      width="0"
      x="162"
      y="80"
    />
    <rect
      height="18"
      rx="0"
      ry="0"
      width="249"
      x="28"
      y="15"
    />
    <rect
      height="0"
      rx="0"
      ry="0"
      width="0"
      x="221"
      y="60"
    />
    <rect
      height="12"
      rx="0"
      ry="0"
      width="249"
      x="28"
      y="50"
    />
    <rect
      height="14"
      rx="0"
      ry="0"
      width="249"
      x="27"
      y="97"
    />
    <rect
      height="11"
      rx="0"
      ry="0"
      width="187"
      x="28"
      y="75"
    />
  </ContentLoader>
</div>
`;
